Configuration/TCA/Overrides/tt_content.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types = 1);
 
-defined('TYPO3_MODE') or die();
+defined('TYPO3') or die();
```

Comment:
Shouldn't we test for both constants as the TYPO3 may not be defined in e.g. TYPO3 10.4?